docs/reference/wattpm/cli-commands.md

```diff
@@ -222,6 +222,46 @@ wattpm applications [id]
 
 - `id` - Process ID or application name (optional if only one app is running)
 
+### `wattpm applications:add`
+
+Adds new applications to a running application from a configuration file.
+
+```bash
+wattpm applications:add [id] <file>
+```
+
+**Arguments:**
+
+- `id` - Process ID or application name (optional if only one app is running)
+- `file` - JSON file containing the applications to add
```

Comment:
what is the format of this file? Why it's plural "applications"? Can it contain more than one?